cmd/dependabot/internal/cmd/test.go

```diff
@@ -88,8 +91,10 @@ func init() {
 	testCmd.Flags().StringVar(&cache, "cache", "", "cache import/export directory")
 	testCmd.Flags().StringVar(&local, "local", "", "local directory to use as fetched source")
 	testCmd.Flags().StringVar(&proxyCertPath, "proxy-cert", "", "path to a certificate the proxy will trust")
+	testCmd.Flags().StringVar(&collectorConfigPath, "collector-config", "", "path to an OpenTelemetry collector config file")
```

Comment:
q: does the `collectorImage` var also need to be added here?

Reply:
No, it's inherited from the root command, like the other images.